db/models.py

```diff
@@ -1 +1,17 @@
 from django.db import models
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return f"Genre: {self.name}"
```

Comment:
Do not add model name in `__str__` since it would be already included in the QuerySet or when you retrieve model
```suggestion
    def __str__(self) -> str:
        return self.name
```